src/Randomizer.Data/Configuration/Yaml/Sassy/items.yml

```diff
@@ -663,6 +663,10 @@
     "20":
     - Text: Okay, you have enough missiles now
     "21":
+    "40":
+    - Text: Someone must have a lot of time on their hands to get all of the missle packs
```

Comment:
Is "missles" here on purpose for pronunciation reasons?

Reply:
I believe so? I noticed the others were like that, so I followed suit. I'm guessing it was originally meant the avoid tracker saying it like miss-aisles. (Honestly we should have a line or two about that - "I hope you don't miss with these miss-aisles" and "tracked more missiles... Or is it miss-aisles? Whatever. Explosive arrows.")